packaging/samba-master.spec.j2

```diff
@@ -598,17 +636,46 @@ Requires: python3-tevent
 Requires: python3-tdb
 Requires: python3-ldb
```

Comment:
May be we can remove above 4 `Requires` as they are provided by the same package.

Reply:
Hm, does that take into consideration internal library builds?

Reply:
OK. I think we need those `Requires` anyhow.